tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -194,4 +194,14 @@ Go To Log Tab And Select A Container
     Click Button    xpath://div[@class="resource-log"]//button[@data-test-id="dropdown-button"]
     Click Button    id=${container_button_id}
     Sleep    4
-    Capture Page Screenshot    logs_page.png
\ No newline at end of file
+    Capture Page Screenshot    logs_page.png
+
+Get Pod Tolerations
+    [Documentation]    Returns the list of pod tolerations
+    [Arguments]    ${pod_name}    ${ns}=rhods-notebooks
+    OpenShiftLibrary.Search Pods    ${pod_name}    namespace=${ns}
+    ${output} =    Run   oc describe pod ${pod_name} -n rhods-notebooks
+    Create File    podspec.txt    ${output}
+    ${tolerations_list} =    Parse File For Tolerations    podspec.txt
```

Comment:
I didn't find were `Parse File For Tolerations` is defined.

You can get a list with the tolerations with this command: 
`oc get pod jupyter-nb-ldap-2dadmin17-0 -n rhods-notebooks -o json | jq '.spec.tolerations[]'`

If you wanted, you could just use a keyword `Pod Should Have Toleration` and run this:
`oc get pod jupyter-nb-ldap-2dadmin17-0 -n rhods-notebooks -o json | jq '.spec.tolerations[]| select(.key == "NotebooksOnly").key'`

Reply:
It's defined in libs/helpers.py